services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/model.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import { useQuery } from 'react-query';
+import { withRouter, Link } from 'react-router-dom';
+import { EditorState, ContentState } from 'draft-js';
+
+import { fetchModel } from '../../../utils/comprehension/modelAPIs';
+import { DataTable, Spinner, TextEditor } from '../../../../Shared/index';
+
+const Model = ({ match }) => {
+  const { params } = match;
+  const { activityId, modelId } = params;
+
+  // const [modelNotes, setModelNotes] = React.useState<string>('');
+  const [errors, setErrors] = React.useState<object>({});
+
+  // cache ruleSets data for handling rule suborder
+  const { data: modelData } = useQuery({
+    queryKey: [`model-${modelId}`, modelId],
+    queryFn: fetchModel
+  });
+
+  // function handleSetModelNotes(text: string){ setModelNotes(text) };
```

Comment:
Commented out code?

Reply:
I opted to leave this in since we're likely adding model notes in the very near future